internal/pkg/errors/error_message.go

```diff
@@ -354,6 +354,7 @@ const (
 	InvalidFilePathErrorMsg            = `invalid file path "%s"`
 	UnsupportedFileFormatErrorMsg      = `unsupported file format for file "%s"`
 	InvalidAlgorithmErrorMsg           = `invalid algorithm "%s"`
+	IncorrectNonceLengthErrorMsg       = `incorrect nonce length from ~/.confluent/config.json passed into encryption`
```

Comment:
Is the course of action to delete this from the config file if this error happens?

Reply:
that will help but i don't really want to explicitly suggest that here, in case user can still correct nonce length, use a new context etc...

Reply:
Ah, ok.

Reply:
How does this happen to users by the way? Does this result from them manually editing the configuration file?

Reply:
yes, or they're mix-using older versions of cli, where cli doesn't have nonce (len == 0)

Reply:
Ah, got it!